allauth/socialaccount/providers/fxa/provider.py

```diff
@@ -1,7 +1,42 @@
+from django.conf import settings
+
 from allauth.socialaccount import providers
 from allauth.socialaccount.providers.base import ProviderAccount
 from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
 
+# https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction#Firefox_Accounts_deployments
+_FXA_DEPLOYMENTS = {
```

Comment:
My thinking was that this configuration:

```python
SOCIALACCOUNT_PROVIDERS = {
    'fxa': {
        'DEPLOYMENT': 'stage',
    }
}
```
was simpler and less error prone than:

```python
SOCIALACCOUNT_PROVIDERS = {
    'fxa': {
        'OAUTH_ENDPOINT': 'https://oauth-stable.dev.lcip.org/v1/',
        'PROFILE_ENDPOINT': 'https://stable.dev.lcip.org/profile/v1/',
    }
}
```
Setting a ``DEPLOYMENT`` automatically picks the base URLs in the style expected by the rest of the code. I left the ``OAUTH_ENDPOINT`` and ``PROFILE_ENDPOINT`` overrides for the the case of running a local FxA server, or if they ever decide to bump that ``v1`` before we can update django-allauth (which could be extended with a ``VERSION`` parameter like the Facebook provider).